snooty/rstspec.toml

```diff
@@ -707,6 +707,26 @@ example = """.. procedure::
 ${0: Steps content}"""
 content_type = "block"
 
+[directive."mongodb:quiz"]
+content_type = "block"
+help = "Embed a quiz with a question argument, using the quizchoice directive to construct answer options."
+example = """.. quiz:: 
+  :quizid: ${1:string}
+   
+   ${2:string}"""
+argument_type = "string"
+options.quizid = {type = "string", required = true}
+
+[directive."mongodb:quizchoice"]
+content_type = "block"
+help = "A possible quiz answer and an explanation to be displayed after submission"
+example = """ .. quizchoice:: ${1:string}
+   :istrue:
```

Comment:
Nit: do we want to perhaps use kebab-case `is-true` like [so](https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L696), or just `true` like [so](https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L314)? I have no personal preferences

Reply:
I personally like `is-true` but open to both! Changing it unless @i80and has any personal preference

Reply:
I like kebab case

Reply:
quiz-id should probably also get the kebab treatment if we do this